doc/folders.md

```diff
@@ -386,3 +391,102 @@ for (BoxItem.Info itemInfo : itemsInFolder) {
     Metadata itemMetadata = itemInfo.getMetadata("properties", "global");
 }
 ```
+
+Create Cascade Policy On Folder
+-------------------------------
+
+To set a metadata policy, which applies metadata values on a folder to new items in the folder, call 
+[`create(BoxAPIConnection api, String folderID, String scope, String templateKey)`][create-policy]
+with the api connection, scope, template key of the metadata template to be cascaded, and the ID of the folder to apply the policy to.
+
+```java
+String folderID = "22222";
+String scope = "enterprise";
+String templateKey = "myTemplate";
+BoxMetadataCascadePolicy.Info metadataCascadePolicyInfo = BoxMetadataCascadePolicy.create(api, folderID, scope, templateKey);
+```
+
+[create-policy]: http://opensource.box.com/box-java-sdk/javadoc/com/box/sdk/BoxMetadataCascadePolicy.html#create-com.box.sdk.BoxAPIConnection-java.lang.String-java.lang.String-java.lang.String
+
+Get a Cascade Policies Information
+----------------------------------
+
+To retrieve information about a specific metadata cascade policy, call 
+[`getInfo()`][get-info]
+
+```java
+String cascadePolicyID = "1234";
+BoxMetadataCascadePolicy metadataCascadePolicy = new BoxMetadataCascadePolicy(api, cascadePolicyID);
+BoxMetadataCascadePolicy.Info metadataCascadePolicyInfo = metadataCascadePolicy.getInfo();
+```
+
+[get-info]: http://opensource.box.com/box-java-sdk/javadoc/com/box/sdk/BoxMetadataCascadePolicy.html#getInfo--
+
+Get All Cascade Policies on Folder
+----------------------------------
+
+To get a list of all cascade policies on a folder, which show the metadata templates that are being applied to all 
+items in the folder, call [`getAll(BoxAPIConnection api, String folderID)`][get-all] with the ID of the folder.
+
+```java
+String folderID = "2222";
+BoxMetadataCascadePolicy.getAll(api, folderID);
+```
+
+You can also call [`getAll(BoxAPIConnection api, String folderID, String enterpriseID, int limit)`][get-all-with-limit] 
+and set the `enterpriseID` option to retrieve only cascade policies owned by a specific enterprise(defaults to the current enterprise).
```

Comment:
We might want to clarify that the method call above only gets cascade policies from the current enterprise; this overload allows getting those from another enterprise